jupyterlab_gcedetails/README.md

```diff
@@ -13,7 +13,8 @@ In order to access the metadata server running on GCE VM, you will need to
 issue the following command to set up port forwarding. This will allow
 requests made locally to port 8889 to be forwarded to a real GCE metadata
 server. Do this in another terminal window since it will create an interactive
-SSH session.
+SSH session. Replace the instance variable in the URL below with the name of an instance 
```

Comment:
Confused about why changes that exist in the upstream are appearing as additions from our fork.